src/dom.js

```diff
@@ -78,7 +78,7 @@ export function waitForBody(doc, callback) {
 
 /**
  * Waits for document's body to be available.
- * @param {!Document} doc
+ * @param {!Document|!ShadowRoot} doc
```

Comment:
Our type check complains about this. Do we call `waitForBodyPromise` with a document instead?

```
src/element-service.js:144: ERROR - actual parameter 1 of module$src$dom.waitForBodyPromise does not match formal parameter
found   : (Document|ShadowRoot)
required: Document
    return dom.waitForBodyPromise(ampdoc.getRootNode()).then(() => {
```